contrib/gce/configure.sh

```diff
@@ -104,6 +104,26 @@ if [ -f "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}" ]; then
   source "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}"
 fi
 
+# cgroupv2 setup
+if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
+  if uname -a | grep -q cos; then
+    if ! grep -q 'systemd.unified_cgroup_hierarchy=true' /proc/cmdline; then
+      echo "Setting up cgroupv2"
+
+      mount_path="/tmp/esp"
+      mkdir -p "${mount_path}"
+      esp_partition="/dev/sda12"
```

Comment:
same question for `sda12` ... is this special?

Reply:
Thanks for the comments. The bootloader files are installed in the ESP (EFI System Partition), which is /dev/sda12 for ChromeOS image

Reply:
yeah, `sda12` is special sauce to modify the GRUB config on COS :)

There is a note about it here as well: https://cloud.google.com/compute/docs/security-bulletins#20190514

Reply:
excellent. thanks @bobbypage @Dragoncell